shared/constants/permissions.ts

```diff
@@ -18,20 +18,32 @@ export const RestrictedMethods = Object.freeze({
 } as const);
 
 ///: BEGIN:ONLY_INCLUDE_IN(flask)
+/**
+ * Exclude permissions by code fencing them to avoid any potential usage of excluded permissions at runtime. See: https://github.com/MetaMask/metamask-extension/pull/17321#pullrequestreview-1287014285.
+ * This is a fix for https://github.com/MetaMask/snaps-monorepo/issues/1103 and https://github.com/MetaMask/snaps-monorepo/issues/990.
+ * TODO: Disable endowment:long-running and eth_account in stable.
```

Comment:
This should be an issue not a TODO ;)

Reply:
What do you mean ? We have https://github.com/MetaMask/snaps-monorepo/issues/1103 and https://github.com/MetaMask/snaps-monorepo/issues/990. This is just a reminder to fence when we ship stable.